media/js/base/core-datalayer.js

```diff
@@ -40,6 +40,13 @@ if (typeof Mozilla.Analytics == 'undefined') {
         return versionResults ? versionResults[1] : null;
     };
 
+    /** Returns latest Fx version.
+    * @return {String} latest Fx version.
+    */
+    analytics.getLatestFxVersion = function() {
+        return document.getElementsByTagName('html')[0].getAttribute('data-latest-firefox');
```

Comment:
Nit: we should probably retrieve the data attribute in the proper way where supported. Seeing as we can use jQuery here `$('html').data('latestFirefox')` should do it, which is shorter too.